Lacrit/assignment3/Dictionary.hpp

```diff
@@ -0,0 +1,42 @@
+#include <fstream>
+#include <iostream>
+#include <unordered_set>
+#include <string>
+
+
+class Dictionary {
+public:
+	Dictionary();
+    Dictionary(std::string);
```

Comment:
Even if nod needed by the compiler, putting names for the parameters helps readers understand what parameters are.